src/DetailsView/Styles/detailsview.scss

```diff
@@ -308,8 +304,18 @@ div.insights-file-issue-details-dialog-container {
     }
     .header-bar {
         padding-left: 12px;
-        position: fixed;
+        position: relative;
         z-index: 1000;
+        display: flex;
```

Comment:
It would be good to remove the redundant `float:` attributes from the direct children, since the flex display will now be laying them out appropriately